requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Remove
-PyYAML==5.1.2
+PyYAML==5.3.1
```

Comment:
We should just phase out `PyYAML` altogether in favor of `raumel.yaml` to avoid two dependencies for similar functionality

Reply:
where do we use PyYAML?

Reply:
Actually, @mza921 it looks like you may have introduced `ruamel.yaml`? Is there a preference over `PyYAML`? I don't have an opinion, but we should standardize on one throughout the project I think.

Reply:
I use `ruamel.yaml` for updating the config file during initial Trakt authorization. Research had led me to [this](https://stackoverflow.com/questions/28557626/how-to-update-yaml-file-using-python). I prefer it over `PyYAML` but didn't bother to update the other modules since I mostly worked on Trakt integration.

Reply:
Ok, let's default to `ruamel.yaml` then and remove `PyYAML`. It looks like it's only used in a couple of places:

- https://github.com/mza921/Plex-Auto-Collections/blob/8d129ad590551ccd1656e07b85d308c8d0514908/app/config_tools.py#L80
- https://github.com/mza921/Plex-Auto-Collections/blob/8d129ad590551ccd1656e07b85d308c8d0514908/app/config_tools.py#L455
- https://github.com/mza921/Plex-Auto-Collections/blob/8d129ad590551ccd1656e07b85d308c8d0514908/app/radarr_tools.py#L1 (looks like this import can just be deleted)